runpod_worker/utils.py

```
@@ -0,0 +1,284 @@
+#!/usr/bin/env python3
+"""
+Utility functions for CharForgex RunPod Worker
+"""
+
+import os
+import sys
+import logging
+import base64
+import tempfile
```

Ruff (0.12.2)

`tempfile` imported but unused

Remove unused import: `tempfile`

(F401)